packages/flutter/lib/src/material/dialog.dart

```diff
@@ -87,6 +90,24 @@ class Dialog extends StatelessWidget {
   /// {@endtemplate}
   final Curve insetAnimationCurve;
 
+  /// {@template flutter.material.dialog.insetPadding}
+  /// The amount of padding added to the outside of the dialog.
```

Comment:
I'm not sure what the right way to describe this is, but I'm assuming this means the padding between the edges of the screen and the dialog itself? I had to think about this for a while and confused between that meaning and the padding between the edges of the dialog and its contents. 

Also, do we only see this when the dialog is expanded to be as large as it can be? If I recall correctly, the dialog will be smaller if there the dialog's contents are small enough

Reply:
Yup. I struggled with that too. I will try come up with a better description.